packages/iov-bns/src/client.ts

```diff
@@ -181,7 +190,9 @@ export class Client implements BcpAtomicSwapConnection {
     return dummyEnvelope(data);
   }
 
-  // getSwapFromState returns all matching swaps that are open (from app state)
+  /**
+   * All matching swaps that are open (from app state)
```

Comment:
Yes, this format does look cleaner.
Always prefix with function name is golang standard, not needed here